staff/sys/migrate-vm

```diff
@@ -45,23 +42,25 @@ from ocflib.misc.shell import red
 
 HostAndVm = namedtuple('HostAndVm', ['host', 'vm'])
 
+client = None
+
 
 def exec_command(cmd, ignore_status_code=False):
-    print(bold(cmd))
     stdin, stdout, stderr = client.exec_command(cmd)
 
-    exit_status = stdout.channel.recv_exit_status()
-    if exit_status != 0 and not ignore_status_code:
-        raise RuntimeError(bold(red('Command exited nonzero ({}): {}'.format(exit_status, cmd))))
+    if not ignore_status_code:
+        exit_status = stdout.channel.recv_exit_status()
+        if exit_status != 0:
+            raise RuntimeError(bold(red('Command exited nonzero ({}): {}'.format(exit_status, cmd))))
 
     return stdin, stdout, stderr
 
 
-def copy_disk_image(vm, num_bytes):
-    _, stdout, _ = client.exec_command('sh -c "dd if=/dev/vg/' + vm + ' bs=32M | pigz"')
+def copy_disk_image(oldpath, newpath, num_bytes):
+    _, stdout, _ = exec_command('sh -c "dd if=' + oldpath + ' bs=32M | pigz"', ignore_status_code=True)
```

Comment:
If `ignore_status_code=False`, then `exec_command` blocks until the command returns, which is not what we want since that would result in a 15GB+ pipe buffer. The status code is ignored in the current script, but it could be checked after the whole transfer finishes.